### PR TITLE
[SALEOR-4568] Extend account register mutation

### PR DESCRIPTION
**Pull request by piotrgrundas**
I want to merge this change because I need that `accountRegister` consumes also first & last name.

[Docs PR](https://github.com/mirumee/saleor-docs/pull/319)

# Impact

* [ ] New migrations
* [x] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [x] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [x] The changes are tested
* [x] GraphQL schema and type definitions are up to date
* [x] Changes are mentioned in the changelog

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-4568 Extend accountRegister mutation](https://app.clickup.com/t/2549495/SALEOR-4568)

File: saleor/graphql/account/mutations/account.py [hunk 1]
```diff
@@ -29,9 +29,19 @@
...
+    language_code = graphene.Argument(
+        LanguageCodeEnum, required=False, description="User language code."
+    )
+
+
+class AccountRegisterInput(AccountBaseInput):
     email = graphene.String(description="The email address of the user.", required=True)
     password = graphene.String(description="Password.", required=True)
+    first_name = graphene.String(description="Given name.")
+    last_name = graphene.String(description="Family name.")
```
Comment (IKarbowiak) [inline saleor/graphql/account/mutations/account.py:44]: You don't need it, as this input object inherits from `AccountBaseInput`.

**Review by IKarbowiak — COMMENTED** [review]
IMO, you should also update tests for `AccountUpdate` as the `AccountInput` has been updated too.

**Review by fowczarek — APPROVED** [review]

**Review by maarcingebala — APPROVED** [review]

File: saleor/graphql/account/mutations/account.py (hunk 1 above)
Comment (piotrgrundas) [reply saleor/graphql/account/mutations/account.py:44]: Oversight :(

**Pull request merged**